autosklearn/automl.py

```diff
@@ -402,7 +402,11 @@ def _fit(
         # == Perform dummy predictions
         num_run = 1
         # if self._resampling_strategy in ['holdout', 'holdout-iterative-fit']:
-        num_run = self._do_dummy_prediction(datamanager, num_run)
+        try:
```

Comment:
This one here is actually not necessary as the failure is written to the log file and this piece of code is built to not fail internally. Could you please undo this change?